### PR TITLE
Make packages deployable and usable for theia

### PR DESCRIPTION
**Pull request by ivy-cst**
But keep vite and jest happy:
- Point the `main` of the `packages.json` to `lib`
- Point the `types` and `module` of `packages.json` to `src`
- Own jest resolver which prefers the `module` (`src`) over `main` (`lib`)
- Vite prefers the `module` (`src`) automatically if it exists

Add a `files` entry to every `package.json`

### CODE REVIEW

**Comment by ivy-cst** [comment]
@ivy-lli If you like it, you can merge it before you make other refactorings. Please test it if your dev workflow still works with this changes!

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "prepare": "yarn icons",
     "ci": "yarn install && yarn icons && yarn type && yarn lint:ci && yarn test:ci && yarn standalone build:production",
     "clean": "lerna run clean",
-    "build": "lerna run build",
+    "build": "lerna run --ignore=@axonivy/inscription-standalone build",
```
Comment (ivy-lli) [inline package.json:13]: no longer needed, right?

File: yarn.lock [hunk 2]
```diff
@@ -5412,6 +5412,19 @@ convert-source-map@^2.0.0:
   resolved "https://registry.yarnpkg.com/convert-source-map/-/convert-source-map-2.0.0.tgz#4b560f649fc4e918dd0ab75cf4961e8bc882d82a"
   integrity sha512-Kvp459HrV2FEJ1CAsi1Ku+MY3kasH19TFykTz2xWmMeq6bk2NU3XXvfJ+Q61m0xktWwt+1HSYf3JZsTms3aRJg==
 
+copyfiles@2.4.1:
```
Comment (ivy-lli) [inline yarn.lock:5415]: I used cpx in the process-editor-client, but this is also not really maintained. 
But there is a fork cpx2 which is. So maybe we should change that in the future, but definitely ok for now.

**Review by ivy-lli — APPROVED** [review]
Seems to work as expected :)

**Pull request merged**